apple/internal/partials/framework_import.bzl

```diff
@@ -131,6 +135,10 @@ def _framework_import_partial_impl(ctx, targets, targets_to_avoid):
         for build_arch in build_archs_found:
             args.add("--slice", build_arch)
 
+        # TODO: Extract `bitcode_strip` into a separate action
```

Comment:
fyi - bundle & sign has historically been a single action because it actually is faster than two actions.  Likewise, things like the thinning of imported frameworks/archives are thinned on import also also speed things up vs. two actions.  On local builds, it doesn't really make a difference, but when you talk about remote builds.  Copying everything to the machine, doing a job, copying them back, then copying things out, doing another job, and copying it back can end up being significantly slower that just doing one set of copies.  Also, the more large copies, the more changes for retransmissions under the hood to the remote host can be required, slowing down the overall build.

Reply:
Thanks for the info. I have always been naïvely thinking that the smaller actions you split the compilation into, the faster incremental build you get. That probably only makes sense for actions that take text files as inputs.

Reply:
Yea, the key is number of inputs/size of inputs/outputs.  The costs can also apply somewhat for local builds, but there it is really the counts that matter since the action needs a setup directory to run in, so the more inputs, the more symlinks that have to be created.